### DISCUSSION TITLE
科普一下，聽聽淡如姐對於數位經濟的看法

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS3073-C/Python-Community-GS3073-C#14] [General]
Podcast也可以聽
網址 https://player.soundon.fm/p/73930beb-4136-4b36-a910-39984335b7bb/episodes/78f71696-c4d2-46a4-beec-537f47927ad3

聽聽淡如姐的看法

**Comment by boson13579** [comment]
聽完Podcast後可以發現，前半段說到關於機器人理財的優點，像人工理財可能會因為人際關西，個人觀點，個人狀態的不同，導致投資人有不必要的損失，機器人可以更理性地去做分析。後半段在講廣告投放，我記得台灣有一間公司Appier就是在做這個的，利用使用者資訊，讓ai分析，用最少的廣告投放成本讓效益最大化。
